registry/remote/repository.go

```diff
@@ -116,6 +119,9 @@ type Repository struct {
 	// referrersState represents that if the repository supports Referrers API.
 	// default: referrersStateUnknown
 	referrersState referrersState
+
+	// referrersTagLocks maps a referrers tag to a lock.
+	referrersTagLocks sync.Map // map[string]sync.Mutex
```

Comment:
Is this a useful comment or commented code? `// map[string]sync.Mutex`

Reply:
It is a comment to show the map structure.